File: day three /javascript/afternoon session /parkingslot.js
function CheckParkingSlot([] parkingSpace[]){
  for (let i = 0; i < parkingSpace.length-1; i++){
        if(parkingSpace[i] == 0)
        return "empty";
      if(parkingSpace[i] == 1)   
    return "occupied";
    }     


function parking(parkingSpace[])     
         for (const i = 0; i <parkingSpace.length-1, i++){
             if(parkingSpace[i] == 0){
            return "this is a free slot";
         }else{
            return "occupied";
          }
      }      return "occupied";

function CarLeavingSlot(parkingSpace [] )

       for (const i = 0; i < 20; i++){
             if(parkingSpace[i].pop()){
            return "car has left the parking space"
        }
    }

module.exports = {CheckParkingSlot, parkingSpace, CarLeavingSlot}

File: day three /python /afternoon session /parking_slot.py
def CheckParkingSlot([] parkingSpace[]):
    for i in range (len(parkingSpace)-1):
        if(parkingSpace[i] == 0):
            return "empty";
        if(parkingSpace[i] == 1):  
            return "occupied";    


def parking(parkingSpace[]):
    for i in range (1,len(parkingSpace)-1):
      if(parkingSpace[i] == 0){
            return "this is a free slot";
         }else{
            return "occupied";
          }
      }      return "occupied";

def CarLeavingSlot(parkingSpace [] ):
    for i in range (1,len(parkingSpace)):
             if(parkingSpace[i].pop()){
            return "car has left the parking space"

